plonk-neon/Cargo.toml

```diff
@@ -0,0 +1,44 @@
+[package]
+name = "plonk_neon"
```

Comment:
Nit: I'd like to rename plonk-neon onto kimchi-neon. Same for plonk-wasm to be consistent.
I'll try to find time in the next few days to propagate this change everywhere.

Reply:
There is a demand from different persons to update the naming conventions because it can become confusing - and it is confusing. `plonk` is too generic.

We will also make new crates/modules for Saffron related definitions, including the o1vm. Therefore, being explicit on what refers to what is important.

> I really don't think that's worth the trouble at this point.
I do not think it would create a huge trouble. It is simply renaming in different places. It can be done as a follow-up, agreed.

Reply:
> Is anyone that's working on o1js complaining about the name?

Yes.